src/components/UNSTABLE__Pagination/_index.scss

```diff
@@ -151,4 +151,20 @@
   &__page-sizer {
     border-right: 1px solid $ui-02;
   }
+
+  &__page-input {
+    height: 100%;
+    justify-content: center;
+    margin-right: $carbon--spacing-03;
+
+    &.#{$prefix}--number input[type='number'] {
+      border: none;
+      background-color: $ui-03;
+    }
+
+    // Visually hide error text for now:
+    .#{$prefix}--form-requirement {
+      @include hidden;
+    }
```

Comment:
The "error message" text is visually hidden for now but I'm unsure what to do with it? 😅 Thoughts?